cmd/zetaclientd-supervisor/lib.go

```diff
@@ -41,7 +41,7 @@ func (w *serializedWriter) Write(p []byte) (n int, err error) {
 	return w.upstream.Write(p)
 }
 
-func getLogger(cfg config.Config, out io.Writer) zerolog.Logger {
+func getLogger(cfg *config.Config, out io.Writer) zerolog.Logger {
```

Comment:
Why? We should avoid using pointers when necessary

Reply:
let me try using non-pointer field. It would need to overwrite every field of a config (except the inner mutex). Let me see if I can remove the mutex.

Reply:
https://github.com/zeta-chain/node/pull/2411/commits/4f61c380bd8629ea044d7284eabf99c09b447131